addons/event/static/src/js/tours/event_tour.js

```diff
@@ -44,22 +44,27 @@ tour.register('event_tour', {
     position: 'bottom',
     width: 175,
 }, {
-    trigger: '.o_event_form_view input[name="name"]',
+    trigger: '.o_event_form_view input[id="name"]',
     content: Markup(_t("This is the <b>name</b> your guests will see when registering.")),
     run: 'text Odoo Experience 2020',
 }, {
-    trigger: '.o_event_form_view input[name="date_end"]',
-    content: Markup(_t("When will your event take place? <b>Select</b> the start and end dates <b>and click Apply</b>.")),
+    trigger: '.o_event_form_view div[name="date_end"]',
+    content: _t("Open date range picker.Pick a Start date for your event"),
```

Comment:
If you are modifying the labels you are going to need to modify the .pot files for this stable version.
Is it really necessary though? We tend to avoid modifying labels in stable.

(Let me know if you are unsure how to do that I can show you).

Reply:
Since I am splitting the step into two 1: picking a start date for event
                                                           2: applying change
It kind of made sense to make change also in labels to describe both steps. The reason behind the step split is the fact that changing values from date-picker into field is not caught by onChange event listener, so now instead of listening to onchange for input we have listener for onClick first on the input field, and secondly on apply button to make sure that information got saved. 
I still think it'd be better idea to make change this way and modify .pot files if necessary tbh, but totally welcome to any new ideas :D